pkg/builder/builder_test.go

```diff
@@ -100,6 +102,62 @@ var _ = Describe("builder", func() {
 		Ω(makeSubPaths(&c)).Should(ContainElement(ContainSubstring("catalog")))
 	})
 
+	It("should enforce volume in vscr", func() {
+		const testVol = "custom-vol"
+		vscr := v1beta1.MakeVscr()
+		vscr.Spec.Volume = &v1.Volume{
+			Name: testVol,
+		}
+		pod := BuildScrutinizePod(vscr)
+		vols := pod.Spec.Volumes
+		Expect(len(vols)).Should(Equal(1))
+		Expect(vols[0].Name).Should(Equal(testVol))
+		cnts := pod.Spec.InitContainers
+		Expect(len(cnts)).Should(Equal(1))
+		volMounts := cnts[0].VolumeMounts
+		Expect(len(volMounts)).Should(Equal(1))
+		Expect(volMounts[0].Name).Should(Equal(testVol))
+
+		cnts = pod.Spec.Containers
+		Expect(len(cnts)).Should(Equal(1))
+		Expect(cnts[0].Name).Should(Equal(names.ScrutinizeMainContainer))
+	})
+
+	It("should add init cnts in vscr to scrutinize pod spec", func() {
+		vscr := v1beta1.MakeVscr()
+		vscr.Spec.InitContainers = []v1.Container{
+			{Name: "init1"},
+			{Name: "init2"},
+		}
+		pod := BuildScrutinizePod(vscr)
+		cnts := pod.Spec.InitContainers
+		Expect(len(cnts)).Should(Equal(3))
+		Expect(cnts[0].Name).Should(Equal(names.ScrutinizeInitContainer))
+		for i := range vscr.Spec.InitContainers {
+			Expect(cnts[i+1].Name).Should(Equal(vscr.Spec.InitContainers[i].Name))
+		}
+	})
+
+	It("should add annotations and labels in vscr to scrutinize pod", func() {
+		vscr := v1beta1.MakeVscr()
+		vscr.Spec.Labels = map[string]string{
+			"label1": "val1",
+			"label2": "val2",
+		}
+		vscr.Spec.Annotations = map[string]string{
+			"ant1": "val3",
+			"ant2": "val4",
+		}
+		pod := BuildScrutinizePod(vscr)
+		verifyLabelsAnnotations := func(objectMeta *metav1.ObjectMeta) {
+			Expect(objectMeta.Labels["label1"]).Should(Equal("val1"))
```

Comment:
Can we use Ω instead of `Expect`. I would like to be consistent in the same file.